<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotsPage</class>
 <widget class="QMainWindow" name="ScreenshotsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="listView">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="WideBar" name="toolBar">
   <property name="windowTitle">
    <string>Actions</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUpload"/>
   <addaction name="actionDelete"/>
   <addaction name="actionRename"/>
   <addaction name="actionView_Folder"/>
  </widget>
  <action name="actionUpload">
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionView_Folder">
   <property name="text">
    <string>View Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WideBar</class>
   <extends>QToolBar</extends>
   <header>ui/widgets/WideBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
